### ISSUE TITLE
git meld master... fails

### ISSUE THREAD

**Issue by wmanley** [wmanley/git-meld#4]
```
git meld master...
```

fails with a merge-base error.  Instead it should be equivalent to:

```
git meld $(git merge-base master HEAD)
```

This is consistent with the behaviour of git diff

**Comment by wmanley** [comment]
It should actually be equivalent to

```
git meld $(git merge-base master HEAD) HEAD
```

i.e. compare to HEAD and not the working directory.  The git diff man page says:

> You can omit any one of <commit>, which has the same effect as using HEAD instead.

I don't particularly like this behaviour but being consistent is probably more important

**Comment by wmanley** [comment]
Fixed in c59a2d147f4091238004bb2f36fe044a62ff5ccc
